openage/launcher/launcher.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020-2020 the openage authors. See copying.md for legal info.
+
+"""
+Backend logic of the openage launcher
+"""
+
+from .mod import ModInfo
+
+LAUNCHER_VERSION = "0.1.0"
+
+
+class LauncherCore:
+    """
+    Holds the launcher status and provides methods for its functionalities
+    exposes an API for the launcher UI
+    """
+    default_path = 'a/path'
+    default_repo = 'a.repo.com'
+    def __init__(self):
+        self.launcher_version = LAUNCHER_VERSION
+        self.version = self.get_version()
+        if self.is_outdated():
+            print('hey, update!')
+
+        self.local_mods = set()
+        self.remote_mods = set()
+
+        self.import_local_mods(self.default_path)
+        self.import_remote_mods(self.default_repo)
+
+        # store uid of enabled mods, ordered by priority
+        self.enabled = []
+
+    @staticmethod
+    def get_version(self):
```

Comment:
`get_version()` should return the version of the launcher. Accessing the engine version should be implemented in its own function `get_engine_version`.